PreviewsSupport/PreviewsSupport/PreviewsSupport.swift

```diff
@@ -105,3 +105,57 @@ extension DefaultPreviewSource: MakeViewControllerProvider where A == UIViewCont
 }
 
 #endif
+
+@available(iOS 18.0, macOS 15.0, tvOS 18.0, watchOS 11.0, *)
+public struct PreviewModifierContextCache {
+ public static var contextCache: [String: Any] = [:]
+}
+
+@available(iOS 18.0, macOS 15.0, tvOS 18.0, watchOS 11.0, *)
+public struct AnyPreviewModifier: PreviewModifier {
```

Comment:
Same with this, does it need to be public?

Reply:
Actually, no